draft-ietf-ppm-dap.md

```diff
@@ -1433,20 +1436,25 @@ never be the case that some reports are completed and others are not.)
 
 The Helper begins an aggregation job when it receives an `AggregationJobInitReq`
 message from the Leader. For each `PrepareInit` conveyed by this message, the
-Helper attempts to initialize VDAF preparation (see {{Section 5.1 of !VDAF}})
-just as the Leader does. If successful, it includes the result in its response
-that the Leader will use to continue preparing the report.
+Helper attempts to asynchronously initialize VDAF preparation (see
+{{Section 5.1 of !VDAF}}) just as the Leader does. If successful, it includes
+the result in its response that the Leader will use to continue preparing the
+report.
 
-To begin this process, the Helper checks if it recognizes the task ID. If not,
-then it MUST abort with error `unrecognizedTask`.
+Upon receipt of an `AggregationJobInitReq`, the Helper checks if it recognizes
+the task ID. If not, then it MUST abort with error `unrecognizedTask`.
 
 Next, the Helper checks that the report IDs in
 `AggregationJobInitReq.prepare_inits` are all distinct. If two preparation
 initialization messages have the same report ID, then the Helper MUST abort with
 error `invalidMessage`.
 
+If the Helper finds the `AggregationJobInitReq` to be valid, it immediately
+responds with HTTP status 201 Created.
```

Comment:
Should this be 202?

Reply:
No, 201 Created is what we use in response to `PUT /tasks/{task-id}/collection_jobs/{collection-job-id}`, so it's appropriate here. 202 Accepted is what we use in response to `GET /tasks/{task-id}/collection_jobs/{collection-job-id}` if the job is not done yet.